### PR TITLE
data submit changes

### PR DESCRIPTION
**Pull request by AndreeaBadiu**

### CODE REVIEW

File: project/src/events/evaluations.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const goToEvaluationsPage = function () {
+    const divEl = document.querySelector('#app');
```
Comment (gcazaciuc) [inline project/src/events/evaluations.js:2]: Consider also cleaning up event listeners with removeEventListener when you navigate from page to page

File: project/src/events/login.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const checkCredentials = function () {
+    const username = document.querySelector('#username').value;
+    const password = document.querySelector('#password').value;
+
+    if (username === 'user' && password === 'pass') {
+        goToEvaluationsPage();
+    } else {
+        alert('Username or Password wrong. Please try again!');
```
Comment (gcazaciuc) [inline project/src/events/login.js:8]: a better user experience is to add a div in the page for errors display and style it with an error class. In which case you can also remove the no-alert from eslint.

File: project/src/views/new-evaluation/newevaluation.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
 const NewEvaluationPage = function () {
     return `
         ${NAV()}
-        <form>
+        <form onSubmit="getFormData()">
```
Comment (gcazaciuc) [inline project/src/views/new-evaluation/newevaluation.js:4]: Attach the submit event with addEventListener

**Review by gcazaciuc — APPROVED** [review]

File: project/src/events/evaluations.js (hunk 1 above)
Comment (AndreeaBadiu) [reply project/src/events/evaluations.js:2]: ok, I will.

File: project/src/events/login.js (hunk 2 above)
Comment (AndreeaBadiu) [reply project/src/events/login.js:8]: I'll try. I've seen eslint doesn't like alerts, but I didn't know why, :D

**Review by s-timofte — APPROVED** [review]

**Pull request merged**